biscuit-auth/src/datalog/symbol.rs

```diff
@@ -282,7 +282,7 @@ impl SymbolTable {
                     crate::token::Scope::Previous => "previous".to_string(),
                     crate::token::Scope::PublicKey(key_id) => {
                         match self.public_keys.get_key(*key_id) {
-                            Some(key) => format!("ed25519/{}", hex::encode(key.to_bytes())),
+                            Some(key) => key.to_string(),
```

Comment:
see comments in the `crypto` mod, I think it is a bit fragile to use `Display` for this